stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -153,15 +153,15 @@ match ret4 {
 
 ### Batch updating data
 
-Following example demonstrates how to insert multiple records with a single INSERT statement executed via the `batchUpdate` operation of the endpoint. This is done by first creating multiple parameter arrays, each representing a single record, and then providing those to the `batchUpdate` operation. Similarly, multiple UPDATE statements could be also executed via `batchUpdate`.
+This example demonstrates how to insert multiple records with a single INSERT statement that is executed via the `batchUpdate` operation of the endpoint. This is done by first creating multiple parameter arrays, each representing a single record, and then passing those arrays to the `batchUpdate` operation. Similarly, multiple UPDATE statements can also be executed via `batchUpdate`.
```

Comment:
Please re-evaluate the usage of commas.

Reply:
The commas look fine.

Reply:
I thought you need a comman before "that is", I guess  you don't.

Reply:
No, you would use a comma before "which", not "that"

Reply:
Got it, thanks.